src/clean.js

```diff
@@ -1,6 +1,6 @@
 // eslint-disable-next-line no-unused-vars
 function clean(ast, newObj, parent) {
-  ['code', 'codeStart', 'loc', 'range'].forEach(name => {
+  ['code', 'codeStart', 'loc', 'range', 'raw'].forEach(name => {
```

Comment:
even here, not sure if this is how we should be dealing w/ this

Reply:
in my opinion, comments shouldn't even be in AST comparison.
Prettier throws an exception already if we failed to print a comment.